src/routes/Things.js

```diff
@@ -47,8 +49,54 @@ let router = express.Router();
  * }
  **/
 
-router.get('/Things', (req, res) => {
-  res.status(200).send();
+const resource = 'Things';
+
+router.get('/', (req, res) => {
+  db().then((dbInstance) => {
+    dbInstance.Things.findAll().then(things => {
+      res.status(200).send(response.generate(things));
+    });
+  });
```

Comment:
Make sure you catch any exception here and in all the other endpoint handlers and create an ApiError with code 500 in that case